3ds/nettle/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+
+pkgname=3ds-nettle
```

Comment:
Just remove the nettle package, it's not necessary for the wslay library. It's only used for examples which won't be built if nettle isn't installed.